<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qq_pcmanagerClass</class>
 <widget class="QWidget" name="qq_pcmanagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qq_pcmanager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(247, 250, 255);
}

QListWidget
{
    background:rgb(104,69,232);   /*表格背景色*/
    color:white;                  /*前景色：文字颜色*/
	font-size:20px;
	border-radius:1px; 
}
 
QListWidget::item:hover
{
    show-decoration-selected:5;
    background:skyblue;
}
 
QListWidget::item:selected
{
    background:lightgray;
    padding:0px;
    margin:0px;
    color:red;
}
 
/*上次选择后保留的状态，鼠标离开后显示*/
QListWidget::item:selected:!active
{
    border-width:0px;
    background:lightgreen;
}</string>
  </property>
  <widget class="QPushButton" name="btn_set">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_min">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>50</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>86</y>
     <width>167</width>
     <height>450</height>
    </rect>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>89</y>
     <width>711</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1">
    <widget class="QPushButton" name="btn_check1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
	background-color:rgb(0, 212, 115);
	color:white;
	font-family: &quot;Microsoft YaHei&quot;;
	font-weight:bold;
	font-size:28px;
    border-radius:6px; 
}

QPushButton:hover
{
    background-color:rgb(0, 232, 155); /*改变背景色*/
    border-style:inset;/*改变边框风格*/
    padding-left:0.3px;
    padding-top:0.3px;
}</string>
     </property>
     <property name="text">
      <string>全面体检</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>409</width>
       <height>145</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page2">
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>409</width>
       <height>145</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page3">
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>409</width>
       <height>145</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page4">
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>409</width>
       <height>145</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page5">
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>409</width>
       <height>145</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>168</width>
     <height>86</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qq_pcmanager.qrc"/>
 </resources>
 <connections/>
</ui>
